nomad/local/integration-tests.nomad

```diff
@@ -174,9 +174,8 @@ job "integration-tests" {
         GRAPL_PLUGIN_REGISTRY_ADDRESS = "http://0.0.0.0:${NOMAD_UPSTREAM_PORT_plugin-registry}"
       }
 
-      # Because Cargo does some... compiling... for some reason.... maybe.....
       resources {
-        memory = 6500
+        memory = 1024
```

Comment:
Recompiling can no longer happen here so I figured I turn it down to what the other tests are using.